html2text/utils.py

```diff
@@ -1,8 +1,17 @@
 import sys
 from html2text import config
+import struct 
 
 from html2text.compat import htmlentitydefs
 
+# Based on http://stackoverflow.com/questions/7105874/valueerror-unichr-arg-not-in-range0x10000-narrow-python-build-please-hel
```

Comment:
Maybe link to the share link for this question instead: http://stackoverflow.com/q/7105874/173630